contribute/developer-guide.md

```diff
@@ -33,61 +33,61 @@ We recommend using the Git command-line interface to download the source code fo
 1. Open a terminal and run `git clone https://github.com/grafana/grafana.git`. This command downloads Grafana to a new `grafana` directory in your current directory.
 1. Open the `grafana` directory in your favorite code editor.
 
-For alternative ways of cloning the Grafana repository, please refer to [GitHub's cloning a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) documentation.
+For alternative ways of cloning the Grafana repository, refer to [GitHub's documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository).
 
-**Warning:** Do not use `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
+> **Caution:** Do not use `go get` to download Grafana. Recent versions of Go have added behavior which isn't compatible with the way the Grafana repository is structured.
 
 ### Configure precommit hooks
 
-We use pre-commit hooks (via [lefthook](https://github.com/evilmartians/lefthook)) to lint, fix, and format code as you commit your changes. Previously the Grafana repository automatically installed these hook when you did `yarn install`, but they are now opt in for all contributors
+We use pre-commit hooks (via [lefthook](https://github.com/evilmartians/lefthook)) to lint, fix, and format code as you commit your changes. Previously, the Grafana repository automatically installed these hook when you ran `yarn install`, but they are now opt-in for all contributors.
 
-Install the lefthook precommit hooks with:
+To install the precommit hooks:
 
 ```sh
 make lefthook-install
 ```
 
-To remove precommit hooks, run
+To remove precommit hooks:
 
 ```sh
 make lefthook-uninstall
 ```
 
-> [!NOTE]
-> Contributors working on the frontend are highly encouraged to install the precommit hooks, even if your IDE formats on save, so the `.betterer.results` file is kept up to sync.
+> We strongly encourage contributors who work on the frontend to install the precommit hooks, even if your IDE formats on save. By doing so, the `.betterer.results` file is kept up to sync.
 
 ## Build Grafana
 
-Grafana consists of two components; the _frontend_, and the _backend_.
+When building Grafana, be aware that it consists of two components: the _frontend_, and the _backend_.
```

Comment:
Maybe format this as a bulleted list with two bullets?

Reply:
done.